lib/perl/Genome/Model/CwlPipeline/Command/Run.pm

```diff
@@ -177,6 +181,103 @@ sub run_cromwell {
     $self->_stage_cromwell_outputs($results_dir);
 }
 
+sub run_cromwell_gcp {
+    my $self = shift;
+    my $yaml = shift;
+    my $results_dir = shift;
+
+    my $workflow_options = Genome::Config::get('cromwell_workflow_options');
+    my $bucket = Genome::Config::get('cromwell_gcp_bucket');
+
+    my $logdir = $self->build->log_directory;
+    my $data_dir = $self->build->data_directory;
+    my $cromwell_url = Genome::Cromwell->server_url;
+    my $queue = Genome::Config::get('lsf_queue_build_worker');
+    my $user_group = Genome::Config::get('lsf_user_group');
+    my $main_workflow_file = $self->build->model->main_workflow_file;
+
+    my $poll_interval_seconds = 30;
+
+    # Cloudize workflow
+    my $cloud_yaml = $yaml; $cloud_yaml =~ s/.ya?ml/_cloud.yaml/;
+    my $guard = Genome::Config::set_env('lsb_sub_additional', 'docker(jackmaruska/cloudize-workflow:1.0.0)');
+    delete local $ENV{BOTO_CONFIG};
+
+    Genome::Sys::LSF::bsub::bsub(
+        queue => $queue,
+        user_group => $user_group,
+        resource_string => 'rusage[mem=512M:internet2_upload_mbps=500]',
+        wait_for_completion => 1,
+        log_file => "$logdir/cloudize_workflow.log",
+        cmd => [
+            "python3",
+            "/opt/scripts/cloudize-workflow.py",
+            $bucket,
+            $main_workflow_file,
+            $yaml,
+            "--output=$cloud_yaml"] );
+
+    $self->debug_message('input yaml: %s', $yaml);
+    $self->debug_message('cloud yaml: %s', $cloud_yaml);
+
+    # Zip dependencies
+    my $deps_zip = "$data_dir/deps.zip";
+    my $prev_dir = getcwd;
+    chdir(dirname($main_workflow_file));
```

Comment:
It might be my preference to use `File::Spec`'s `splitpath` instead of relying on `dirname`, but I concede that this is probably fine here 😄